src/components/Navbar/Notifica/index.js

```diff
@@ -39,32 +76,46 @@ export default class Notifica extends React.PureComponent {
 
     // TODO: use request's shouldPoll once it is usable
     componentDidMount() {
-        this.interval = setInterval(
-            () => {
-                const { notificationCountRequest } = this.props;
-                notificationCountRequest.do();
-            },
-            8000,
-        );
+        this.startRequest();
     }
 
     componentWillUnmount() {
-        clearInterval(this.interval);
+        clearTimeout(this.timeout);
+    }
+
+    startRequest = () => {
+        const { notificationsCountRequest } = this.props;
```

Comment:
To inject parameters at constructor use:
```
        props.setDefaultRequestParams({
            ...
        });
```
To start request at start, use ```onMount``` params